client/src/components/Home/HomeGridCurrentGlmLock/ModalLockGlm/LockGlmNotification/LockGlmNotification.tsx

```diff
@@ -53,11 +53,7 @@ const LockGlmNotification: FC<LockGlmNotificationProps> = ({
       isVertical
     >
       <div className={styles.notification}>
-        <Svg
-          classNameWrapper={styles.iconWrapper}
```

Comment:
Usage of class is removed, but not the class?

Reply:
It was use of non-existent class.